packages/components/src/styles/common-styles.spec.ts

```diff
@@ -1,23 +1,32 @@
 import type { JssStyle } from 'jss';
-import type { PropertiesHyphen } from 'csstype';
 import {
   addImportantToEachRule,
   addImportantToRule,
   focusPseudoJssStyle,
   getBackfaceVisibilityJssStyle,
+  getFrostedGlassBackgroundJssStyles,
   getHiddenTextJssStyle,
   getInsetJssStyle,
   getTransition,
   pxToRemWithUnit,
 } from './common-styles';
 
 describe('getTransition()', () => {
-  it.each<[keyof PropertiesHyphen, string]>([
-    ['color', 'color var(--p-transition-duration, .24s) ease'],
-    ['box-shadow', 'box-shadow var(--p-transition-duration, .24s) ease'],
-  ])('should for %o return %o', (cssProperty, expected) => {
-    expect(getTransition(cssProperty)).toBe(expected);
-  });
+  it.each<Parameters<typeof getTransition>>([
+    ['color', undefined, undefined, undefined],
+    ['box-shadow', 'motionDurationShort', 'motionEasingBase', 'motionDurationShort'],
+    ['color', 'motionDurationModerate', 'motionEasingIn', 'motionDurationModerate'],
+    ['box-shadow', 'motionDurationLong', 'motionEasingOut', 'motionDurationLong'],
+
+    ['color', 'motionDurationVeryLong', 'linear', 'motionDurationVeryLong'],
+
+    ['box-shadow', undefined, 'none', undefined],
+  ])(
+    'should return correct style for for cssProperty: %s, duration: %s, ease: %s and delay',
+    (cssProperty, duration, ease, delay) => {
+      expect(getTransition(cssProperty, duration, ease, delay)).toMatchSnapshot();
```

Comment:
Could use rest and spread operator `...params`.  
Why are these snapshot tests now when they were static tests before?

Reply:
because the result is really long and the test is not easy to read.